app/presenters/collection_presenter.rb

```diff
@@ -90,7 +95,17 @@ def created_datetime
   end
 
   def contact_emails
-    contact_emails_attributes.map(&:email).join(', ')
+    collection_form.contact_emails.map(&:email).join(', ')
+  end
+
+  def related_links
+    return ['None provided'] if collection_form.related_links.blank?
+
+    collection_form.related_links.filter_map do |related_link|
+      next if related_link.url.blank?
+
+      link_to_new_tab(related_link.text || related_link.url, related_link.url)
+    end
   end
```

Comment:
This method moved from the form presenter into the collection presenter because works no longer have related links.